src/backend/distributed/operations/repair_shards.c

```diff
@@ -1471,7 +1467,7 @@ RecreateTableDDLCommandList(Oid relationId)
 		appendStringInfo(dropCommand, DROP_REGULAR_TABLE_COMMAND,
 						 qualifiedRelationName);
 	}
-	else if (relationKind == RELKIND_FOREIGN_TABLE)
+	else if (IsForeignTable(relationId))
 	{
```

Comment:
Don't we have any DROP FOREIGN TABLE regression test? Why is this line doesn't have coverage do you think?